src/Sentry.Maui/Internal/SentryMauiInitializer.cs

```diff
@@ -15,6 +15,12 @@ public void Initialize(IServiceProvider services)
         // Initialize the Sentry SDK.
         var disposable = SentrySdk.Init(options);
 
+        if (options.AttachScreenshots)
+        {
+            SentrySdk.ConfigureScope(s =>
```

Comment:
Does this mean if I call `scope.Clear()` the screenshot is gone? I believe we had issues with that on Unity cc @bitsandfoxes

Reply:
Modified to use Hint so it isn't cleared any more.

Reply:
`SentryMauiScreenshotProcessor` is in the scope, if I `scope.Clear()` happens the processor is gone

Reply:
Should be resolved now.

Reply:
You did so by adding to the options instead? (a clarification on how you resolves (as opposed to "should be resolved") helps the reviewer understand how.

I see this is added to both the scope and the options now, is this intended? Are we taking 2 screenshots possibly?

Reply:
Yeah, I forgot to remove it from scope :)